app/services/analytics_events.rb

```diff
@@ -163,6 +163,16 @@ def authentication_confirmation_reset
     track_event('Authentication Confirmation: Reset selected')
   end
 
+  # @param [Date] verified_at
+  # Tracks when a profile is automatically rejected due to being under review for 30 days
+  def automatic_fraud_rejection(verified_at:, **extra)
+    track_event(
+      'Automatic Fraud Rejection',
```

Comment:
Could we add a descriptor or namespace to contextualize this to identity proofing, since "fraud" could apply to a lot of different actions in the application. Maybe `IdV` like we use elsewhere?

```suggestion
      'IdV: Automatic Fraud Rejection',
```

Similar note could apply to the naming of the `FraudRejectionDailyJob` class.

Reply:
Maybe `Fraud: `?